plugins/view-webhook.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: view-webhook
+spec:
+  version: v0.0.3
+  homepage: https://github.com/Trendyol/kubectl-view-webhook
+  shortDescription: Visualize your webhook configurations in Kubernetes.
```

Comment:
`Visualize your webhook configurations` is sufficient. It's clear kubectl plugins are for Kubernetes. :)

Reply:
Thanks to your valuable comments, done btw. 🙏